extensions/amp-ad/0.1/amp-ad.css

```diff
@@ -67,9 +67,12 @@ amp-embed iframe {
   border-radius: 5px;
 }
 
-amp-ad[type=adsense] > iframe {
+amp-ad[data-a4a-upgrade-type="amp-ad-network-doubleclick-impl"] > iframe,
```

Comment:
Are we sure we want to remove amp-ad[type=adsense] > iframe here?  That would have applied to both delayed and fast fetch for Adsense.  Instead lets make this the following which will ensure centering for all adsense and only FF for doubleclick:

amp-ad[type=adsense] > iframe,
amp-ad[type=amp-ad-network-doubleclick-impl] > iframe